exercises/hamming/src/example.c

```diff
@@ -4,13 +4,13 @@
 
 size_t compute(const char *lhs, const char *rhs)
 {
-   size_t lhs_len = strlen(lhs);
-   size_t rhs_len = strlen(rhs);
-   size_t count = 0;
-   size_t i = 0;
-   for (i = 0; i < lhs_len && i < rhs_len; ++i) {
-      if (lhs[i] != rhs[i]) {
-         ++count;
+   size_t count = -1;
+   if (lhs && rhs && (strlen(lhs) == strlen(rhs))) {
+      count = 0;
+      for (size_t i = 0; i < strlen(lhs); ++i) {
```

Comment:
One final nitpick: `strlen` isn't a cheap operation, so doing it each time through the `for` loop is going to be O(n ^ 2) (that is, runtime will grow in proportion to the square of the length of the string). It would be better to calculate it once at the beginning and reuse that pre-calculated value.

Reply:
I'll revert.